### DISCUSSION TITLE
Scroll slides via scroll event

### DISCUSSION THREAD

**Discussion by nfiacco** [davidjerleke/embla-carousel#636] [Help] [answered]
Is there a way to use scroll to navigate through slides?

**Comment by davidjerleke** [comment]
Hi @nfiacco,

Thank you for your question. Sorry, I’m not following. Care to explain further?

Best,
David

**Reply by nfiacco** [reply]
For example, by scrolling the horizontal mousewheel the slides would also scroll

**Reply by davidjerleke** [reply, marked as answer]
@nfiacco I think you’re looking for the wheel gesture [plugin](https://www.embla-carousel.com/plugins/wheel-gestures/).

Best,
David

**Reply by nfiacco** [reply]
Amazing, thank you!
